Update Readme.md (#1)
DevExpress-Examples/how-to-create-a-custom-save-button-in-aspxrichedit-with-automatic-enabling-disabling-t576860: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T576860)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Rich Text Editor for ASP.NET Web Forms - How to automatically change the availability of a custom Save button

> This example demonstrates how to create a custom **Save** button that is disabled when the document has no changes; ans is enabled when the document has been changed.

Reviewer: ```suggestion
This example demonstrates how to create a custom **Save** button that is disabled when the document has no changes and is enabled when the document is changed.
```

Proofread

## Implementation Details

> 1. Create a function where call the [FileSaveCommand.getState](https://docs.devexpress.com/AspNet/js-FileSaveCommand.getState) method to get the state of the default **Save** button.

Reviewer: ```suggestion
1. Create a function where you call the [FileSaveCommand.getState](https://docs.devexpress.com/AspNet/js-FileSaveCommand.getState) method to get the state of the default **Save** button.
```
